build.sh

```diff
@@ -10,27 +10,6 @@ TOOLS_DIR=$SCRIPT_DIR/tools
 NUGET_EXE=$TOOLS_DIR/nuget.exe
 CAKE_EXE=$TOOLS_DIR/Cake/Cake.exe
 
-# Define default arguments.
-SCRIPT="build.cake"
```

Comment:
Aaaah, I confused this file with the one in the example repo, https://github.com/cake-build/example/blob/master/build.sh. Sorry! Should I move the PR to https://github.com/cake-build/example?